### PR TITLE
Update tokenizer.py

### PR DESCRIPTION
**Pull request by akshatrocky**
Added proper tokenizer support for Hindi Language which would prevent crash while fine tuning Hindi language.

### CODE REVIEW

File: TTS/tts/layers/xtts/tokenizer.py [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import logging
 import os
 import re
 import textwrap
 from functools import cached_property
 
+import pypinyin
 import torch
+from hangul_romanize import Transliter
+from hangul_romanize.rule import academic
```
Comment (eginhard) [inline TTS/tts/layers/xtts/tokenizer.py:9]: These and other changes not related to Hindi should be reverted.

**Review by eginhard — CHANGES_REQUESTED** [review]
Thanks a lot for the PR, this is great!

It includes some changes that are not related to Hindi because there are some small differences in the code between this fork and the original repo and it looks like you started from the original one. Could you revert these unrelated changes?

**Review by akshatrocky — COMMENTED** [review]
Changes fixed, pls check.

**Comment by manash997** [comment]
Hi @akshatrocky i am getting this error, while trying to run tokenizer script from your branch:
tokenizer.py", line 806, in test_expand_numbers_multilingual
    assert out == b, f"'{out}' vs '{b}'"
AssertionError: 'Через двенадцать целых пять десятых секунды.' vs 'Через двенадцать запятая пять секунды.'

**Comment by akshatrocky** [comment]
Which language are you trying to generate @manash997 ?

**Comment by manash997** [comment]
> Which language are you trying to generate @manash997 ?
> 
> (Edit) - How are you running the tokenizer script, from fine tuning XTTS?

i just tried running it as a standalone python script. The changes for Hindi, look good to me. However wanted to test the entire script once.

**Comment by eginhard** [comment]
Yes, the checks in this file are not actually run during any tests and were probably already broken for a while. Otherwise everything looks good. The author deleted their account, so I had to recreate the PR in #64.

**Pull request closed without merging**